src/main/java/mate/academy/spring/mapper/impl/request/OrderRequestMapper.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.mapper.impl.request;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoRequestMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.request.OrderRequestDto;
+import mate.academy.spring.service.UserService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderRequestMapper implements DtoRequestMapper<OrderRequestDto, Order> {
+    private final UserService userService;
+
+    public OrderRequestMapper(UserService userService) {
+        this.userService = userService;
+    }
+
+    @Override

```

Comment:
`new User(dto.getUserId())` is enought
```suggestion
public class OrderRequestMapper implements DtoRequestMapper<OrderRequestDto, Order> {
    @Override
```
![image](https://user-images.githubusercontent.com/20326858/230592183-0d3b4986-c8ef-4220-89f9-b356ce37492e.png)